plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginWithMinifiedLibsTest.kt

```diff
@@ -23,6 +23,7 @@ class SentryPluginWithMinifiedLibsTest :
               implementation 'com.google.android.gms:play-services-mlkit-text-recognition:18.0.0'
               implementation 'com.adcolony:sdk:4.7.1'
               implementation 'com.appboy:android-sdk-ui:19.0.0'
+              implementation files("testlib/stripe-problematic.jar")
```

Comment:
Can we actually add a dependency directly to the stripe lib that is problematic?

Reply:
Yes, can update